### PR TITLE
hide file settings link

### PR DESCRIPTION
**Pull request by fracado**

### CODE REVIEW

**Review by matsaur — COMMENTED** [review]
For me this solution is a little bit too "generic". What do you think about using data attribute? (data-cy-files-navigation-settings-button)

File: core/css/server.css [hunk 1]
```diff
@@ -42,3 +42,7 @@ ionos-user-menu:not(:defined) {
 header#header {
 	margin-top: 64px !important;
 }
+
```
Comment (bromiesTM) [inline core/css/server.css:45]: suggestion(non-blocking): we should add some kind of comment on why we are doing this. We probably won't remember at some point in the future

File: core/css/server.css [hunk 2]
```diff
@@ -42,3 +42,7 @@ ionos-user-menu:not(:defined) {
...
+.app-navigation .app-navigation-entry__settings li:nth-child(2) {
```
Comment (bromiesTM) [inline core/css/server.css:46]: I second that → https://github.com/IONOS-Productivity/nc-ionos-theme/pull/14#pullrequestreview-2260612557

File: core/css/server.css [hunk 3]
```diff
@@ -42,3 +42,7 @@ ionos-user-menu:not(:defined) {
...
+	display: none;
```
Comment (bromiesTM) [inline core/css/server.css:47]: question: did you check if we are able to set the default values of the settings via configs etc?

**Review by bromiesTM — CHANGES_REQUESTED** [review]
Review OK with Minor amends

File: core/css/server.css (hunk 3 above)
Comment (fracado) [reply core/css/server.css:47]: I thought, as the settings aren't affected by this change, the default settings remain as we want them. And therefore nothing needs to be done there. But please correct me if I'm wrong.

File: core/css/server.css (hunk 1 above)
Comment (fracado) [reply core/css/server.css:45]: done with 6479336812e5ab993c901470f787c779ed05e7e0

File: core/css/server.css (hunk 2 above)
Comment (fracado) [reply core/css/server.css]: done with 6479336812e5ab993c901470f787c779ed05e7e0

File: core/css/server.css (hunk 3 above)
Comment (bromiesTM) [reply core/css/server.css]: I double-checked, and it seems like hidden files should be enabled by default, but currently isn't.

**Pull request merged**